pkg/components/prometheus-operator/component.go

```diff
@@ -67,20 +69,36 @@ type Prometheus struct {
 	ExternalURL                 string            `hcl:"external_url,optional"`
 }
 
+// Operator object collects sub component Prometheus operator related information.
+type Operator struct {
+	NodeSelector                   map[string]string `hcl:"node_selector,optional"`
+	Tolerations                    []util.Toleration `hcl:"tolerations,block"`
+	TolerationsRaw                 string
+	AdmissionWebhookTolerations    []util.Toleration `hcl:"admission_webhook_tolerations,block"`
+	AdmissionWebhookTolerationsRaw string
+}
+
+// AlertManager object collects sub component AlertManager related information.
+type AlertManager struct {
+	Retention      string            `hcl:"retention,optional"`
+	ExternalURL    string            `hcl:"external_url,optional"`
+	Config         string            `hcl:"config,optional"`
+	NodeSelector   map[string]string `hcl:"node_selector,optional"`
+	Tolerations    []util.Toleration `hcl:"tolerations,block"`
+	TolerationsRaw string
+	StorageSize    string `hcl:"storage_size,optional"`
+}
+
 type component struct {
 	Grafana *Grafana `hcl:"grafana,block"`
 
 	Namespace string `hcl:"namespace,optional"`
 
-	PrometheusOperatorNodeSelector map[string]string `hcl:"prometheus_operator_node_selector,optional"`
```

Comment:
Can introduce new block in separate commit from adding new fields? That will make reviewing easier.